### PR TITLE
add(docs): merge.automerge_dependencies recipe and config docs

### PR DESCRIPTION
**Pull request by chdsbd**
This change documents the newly released `merge.automerge_dependencies` configuration option.

### CODE REVIEW

File: docs/docs/recipes.md [hunk 1]
```diff
@@ -19,7 +19,32 @@ require_automerge_label = false # default: true
...
+# .kodiak.toml
+[merge]
+automerge_label = "ship it!"
+
+[merge.automerge_dependencies]
+# auto merge all PRs opened by "dependabot" that are "minor" or "patch" version upgrades. "major" version upgrades will be ignored.
+versions = ["minor", "patch"]
+usernames = ["dependabot"]
+
+# if using `update.always`, add dependabot to the blacklist to allow
```
Comment (sbdchd) [inline docs/docs/recipes.md:43]:
```suggestion
# if using `update.always`, add dependabot to `update.ignore_usernames` to allow
```

**Review by sbdchd — APPROVED** [review]

**Comment by staabm** [comment]
This looks awesome. Exactly what I was searching for lately. Will give it a shot next week

File: docs/docs/config-reference.md [hunk 2]
```diff
@@ -48,6 +48,54 @@ Require that the automerge label (`merge.automerge_label`) be set for Kodiak to
 
 When disabled, Kodiak will immediately attempt to merge any PR that passes all GitHub branch protection requirements.
 
+### `merge.automerge_dependencies.versions`
```
Comment (staabm) [inline docs/docs/config-reference.md:51]: Should this setting mention, that the regular rules apply, like a green build on required checks etc?

Comment (chdsbd) [reply docs/docs/config-reference.md:51]: good idea, I've updated to docs to mention that pull requests must pass GitHub branch protections rules to be merged.

**Pull request merged**